rasa/core/domain.py

```diff
@@ -815,6 +806,7 @@ def as_dict(self) -> Dict[Text, Any]:
             KEY_RESPONSES: self.templates,
             KEY_ACTIONS: self._custom_actions,  # class names of the actions
             KEY_FORMS: self.forms,
+            KEY_E2E_ACTIONS: self.action_texts,
```

Comment:
this means that Rasa X will display / dump `action_texts`. We need to remove this key in `cleaned_domain`

Reply:
done in `cleaned_domain`!